package.json

```diff
@@ -91,6 +91,7 @@
   },
   "scripts": {
     "start": "node tools/proxy/proxy.js",
+    "stop": "killall java && killall node",
```

Comment:
This seems to be too much of a bazooka. If the developer has other software running via java or node, those will all get killed (for example, a java-based IDE ^^). You could try to do  get the PID which runs a specific port via `lsof -i :9000 -Fp`, but the output still needs to be parsed. I suggest to remove this bit from the PR for now, so that it can get merged.

Reply:
Alrighty, I get what you mean . Then I keep this bazooka to myself 💥

Reply:
Trouble on my end got fixed with 5253e65. The ... operator seems to have been the cause :)

Reply:
The explaination for this is that the backend starts the node script with a library that can start a own nodejs process in a defined environment. But the nodejs support seems to be not supported / maintained since a quite some time and therefore the newest nodejs version seems to be `0.12.<something>` and this is far too old 🧓. Nodejs only supports the spread operator (...) since version `8.*`. Thus I took merged the objects to form the env vars the complicated way 🤷‍♂️ . This seems to work and fixes the diffing for MacOS.